### DISCUSSION TITLE
Cargo console needs a UI overhaul

### DISCUSSION THREAD

**Discussion by SweptWasTaken** [space-wizards/space-station-14#4035] [General]
![113181990-62da1400-9220-11eb-95b1-de045e9bc91b](https://user-images.githubusercontent.com/49448379/119011401-90238280-b984-11eb-9fb6-67bf04171f7f.png)

**Comment by PaulRitter** [comment]
maybe also create the new design with an additional tab for centcomm orders (stuff you can sell to centcomm) in mind
